src/Button/Button.js

```diff
@@ -22,15 +22,15 @@ function Button(props, ref) {
   return (
     <button
       className={classNames(
-        className,
         'relative uppercase focus:outline-none h-12 px-2',
-        `${background} ${borderWidth} ${colors.border}`
+        `${background} ${borderWidth} ${colors.border}`,
+        className
       )}
       {...mergeProps(buttonProps, hoverProps, focusProps)}
       disabled={disabled}
     >
       {typeof children === 'string' ? (
-        <Text className="text-lg font-normal">{children}</Text>
+        <Text variant="h4">{children}</Text>
```

Comment:
I'm not 100% sure if it's correct to use semantic html such as headers inside of buttons

Reply:
Ah, well that's good to know. If that's true I think I'll need to make bodyLarge Poppins then.